src/linux/classes/com/alibaba/wisp/engine/WispTask.java

```diff
@@ -606,6 +610,17 @@ void countExecutionTime(long beginTime) {
         return this.threadWrapper.getStackTrace();
     }
 
+    long getCpuTime() {
+        // concurrent inaccurate is acceptable
+        if (jvmParkStatus == WAITING || jdkParkStatus == WAITING) {
+            assert totalTs != 0;
+            return totalTs;
+        } else {
+            assert enterTs != 0;
```

Comment:
clean enterTs at scchedule() and :
```
long enter = enterTs;
return totalTs + (enterTs == 0 ? 0 : System.nanoTime() - enterTs) 
```

Reply:
long enter = enterTs;
return totalTs + (enter == 0 ? 0 : System.nanoTime() - enter)

Reply:
OK.